### ISSUE TITLE
Sort by status, title, priority

### ISSUE THREAD

**Issue by alavatelliMCA** [coyote-team/coyote#3]
Sort by status, title, priority
Sort by function available on alternate views too

**Comment by seeReadCode** [comment]
I'm separating the second request into another ticket.

**Comment by seeReadCode** [comment]
Status sort depends on #6

**Comment by seeReadCode** [comment]
At this point, title and priority are deployable
